graphstorm-processing/graphstorm_processing/graph_loaders/row_count_utils.py

```diff
@@ -184,8 +184,13 @@ def _add_counts_for_graph_structure(
         all_entries_counts = []  # type: List[Sequence[int]]
         for type_value in self.metadata_dict[edge_or_node_type_key]:
             logging.info("Getting counts for %s, %s", top_level_key, type_value)
-            relative_file_list = self.metadata_dict[top_level_key][type_value]["data"]
+            # Get the data dictionary for this type and create a copy
+            # to avoid modifying shared references
+            type_data_dict = self.metadata_dict[top_level_key][type_value].copy()
```

Comment:
What is the estimated size for the ``type_data_dict`` here? I suppose it should be meta information so relatively small.

Reply:
Yeah up to a a couple of thousand strings, so a few KB-MB.